sunpy/instr/tests/test_goes.py

```diff
@@ -234,16 +236,17 @@ def test_calculate_radiative_loss_rate():
     # Check function gives correct results.
     # Test case 1: GOESLightCurve object with only flux data
     goeslc_test = goes.calculate_radiative_loss_rate(goeslc_input)
-    goeslc_expected = goes.calculate_temperature_em(goeslc_input)
-    goeslc_expected.data["rad_loss_rate"] = \
-      np.array([5.44914366e+19, 5.44914366e+19, 5.43465905e+19,
-                5.38282295e+19, 5.42019309e+19])
-    assert_frame_equal(goeslc_test.data, goeslc_expected.data)
+    exp_data = np.array([1.78100055e+19, 1.66003113e+19, 1.71993065e+19,
+                         1.60171768e+19, 1.71993065e+19])
+    np.testing.assert_allclose(goeslc_test.data.rad_loss_rate[:5],
+                               exp_data)
 
     # Test case 2: GOESLightCurve object with flux and temperature
     # data, but no EM data.
     goes_test = goes.calculate_radiative_loss_rate(goeslc_no_em)
-    assert_frame_equal(goeslc_test.data, goeslc_expected.data)
+    # we test that the column has been added
+    assert "rad_loss_rate" in goes_test.columns
```

Comment:
Why do we only test the column is added here and not that the values are correct?

Reply:
done using pytest-arraydiff